src/dcat-ap/dcat-dataset.ts

```diff
@@ -84,7 +84,7 @@ export class DcatDataset {
 
   get isFeatureLayer (): boolean { return /_/.test(this.id); }
   get hasGeometryType (): boolean { return !!this._get('geometryType'); }
-  private get _supportedExtensions () { return _.get(this._dto, 'server.supportedExtensions'); }
+  private get _supportedExtensions () { return _.get(this._dto, 'supportedExtensions'); }
```

Comment:
lol `this._dto` 😂 I'm loving these names!

Reply:
https://en.wikipedia.org/wiki/Data_transfer_object 🙂